nilearn/plotting/surface/tests/test_plotly_backend.py

```diff
@@ -158,27 +149,77 @@
 ]
 
 
+@pytest.fixture
+def plotly_backend():
+    return PlotlySurfaceBackend()
+
+
+def test_configure_title():
+    """Test nilearn.plotting.surface._plotly_backend._configure_title."""
+    assert _configure_title(None, None) == {}
+    assert _configure_title(None, 22) == {}
+    config = _configure_title("Test Title", 22, color="green")
+    assert config["text"] == "Test Title"
+    assert config["x"] == 0.5
+    assert config["y"] == 0.96
+    assert config["xanchor"] == "center"
+    assert config["yanchor"] == "top"
+    assert config["font"]["size"] == 22
+    assert config["font"]["color"] == "green"
+
+
 @pytest.mark.parametrize("full_view", EXPECTED_CAMERAS_PLOTLY)
-def test_get_view_plot_surf(full_view):
-    """Test if nilearn.plotting.surface._plotly_backend._get_view_plot_surf
```

Comment:
I've split this test into 3 separate tests and added a case to increase coverage.